src/db/update/ExcludeList.cxx

```diff
@@ -74,17 +74,12 @@ ExcludeList::Check(Path name_fs) const noexcept
 		}
 	}
 
-	for (const auto &i : patterns) {
-		try {
-			if (i.Check(NarrowPath(name_fs).c_str()))
-				return true;
-		} catch (...) {
-		}
```

Comment:
You removed the try/catch. Why?

Reply:
Wasn't sure about this. I assume it would have to be included in the lambda.

Reply:
I don't understand what you assumed. The try/catch is still missing, making this function crash on error.

Reply:
Got rid of the conversion. It makes no sense honestly.